ckan/public/base/javascript/resource.config

```diff
@@ -20,49 +19,4 @@ apply_html_class =
 [groups]
 
 ckan =
-    modules/select-switch.js
-    modules/slug-preview.js
-    modules/basic-form.js
-    modules/confirm-action.js
-    modules/api-info.js
-    modules/autocomplete.js
-    modules/custom-fields.js
-    modules/data-viewer.js
-    modules/table-selectable-rows.js
-    modules/resource-form.js
-    modules/resource-upload-field.js
-    modules/resource-reorder.js
-    modules/resource-view-reorder.js
-    modules/follow.js
-    modules/activity-stream.js
-    modules/dashboard.js
-    modules/resource-view-embed.js
-    view-filters.js
-    modules/resource-view-filters-form.js
-    modules/resource-view-filters.js
-    modules/table-toggle-more.js
-    modules/dataset-visibility.js
-    modules/media-grid.js
-    modules/image-upload.js
-    modules/followers-counter.js
-
 main =
-    apply_html_class
-    plugins/jquery.inherit.js
-    plugins/jquery.proxy-all.js
-    plugins/jquery.url-helpers.js
-    plugins/jquery.date-helpers.js
-    plugins/jquery.slug.js
-    plugins/jquery.slug-preview.js
-    plugins/jquery.truncator.js
-    plugins/jquery.masonry.js
-    plugins/jquery.form-warning.js
-    plugins/jquery.images-loaded.js
-
-    sandbox.js
-    module.js
-    pubsub.js
-    client.js
-    notify.js
-    i18n.js
-    main.js
```

Comment:
we need to keep these empty files for now?

Reply:
Yep. They are required for backward compatibility with extensions that depends on these files. It may look strange, why I removed all entries from them, but it's quite simple - if you leave them unchanged and some extension depends on them, two identical versions of javascript files will be included into page(one from web assets and one compiled as extension dependency) and this leads us to numerous jQuery conflicts and such kind of bugs.